.github/workflows/build-and-test.yml

```diff
@@ -65,14 +65,21 @@ jobs:
       - name: Gradle build
         run: ./gradlew build -x check -x docs-website:build
       - name: Smoke test
-        run: |
```

Comment:
@hsheth2 was this intentional?

Reply:
Yep - smoke.sh now runs the up command within the script